tests/wallet/clawback/test_clawback_lifecycle.py

```diff
@@ -195,9 +195,10 @@ async def test_clawback_spends(self, cost_logger: CostLogger) -> None:
                 cost_logger=cost_logger,
                 cost_log_msg="Create Second Clawback",
             )
-
-            new_cb_coin = (await sim_client.get_coin_records_by_puzzle_hash(cb_puz_hash))[1].coin
-
+            cb_coins = await sim_client.get_coin_records_by_puzzle_hash(cb_puz_hash)
+            new_cb_coin = cb_coins[0].coin
+            if cb_coins[0].spent:
+                new_cb_coin = cb_coins[1].coin
```

Comment:
please add a comment to what this logic intends to do, and maybe why it's safe to assume that the second coin in this list is the one you want

Reply:
I also wonder, whats the purpose of this change?